monai/apps/datasets.py

```diff
@@ -223,6 +227,8 @@ class DecathlonDataset(Randomizable, CacheDataset):
             may set `copy=False` for better performance.
         as_contiguous: whether to convert the cached NumPy array or PyTorch tensor to be contiguous.
             it may help improve the performance of following logic.
+        runtime_cache: whether to compute cache at the runtime, default to `False` to prepare
+            the cache content at initializaiton.
```

Comment:
We can (optionally) provide more information here. 

When using this option in multi-gpu distributed training, `torch.cuda.set_device(local_rank)` must be called before initializing this class.

Reply:
Sure, updated in the latest commit.

Thanks.